src/pages/docs/chat/react-ui-components/components.mdx

```diff
@@ -122,6 +122,9 @@ The `App` component is a complete, ready-to-use chat application that combines a
 - Provides customizable width and height
```

Comment:
On the main landing page for UI components (index.mdx ? ) I think we should have an image as well, as the landing page is the showcase